start.js

```diff
@@ -31,29 +30,11 @@ async function collectDataUsage () {
   }
 }
 
-function createSessionDataDefaults () {
-// Create template session data defaults file if it doesn't exist
-  const dataDirectory = path.join(projectDir, '/app/data')
-  const sessionDataDefaultsFile = path.join(dataDirectory, '/session-data-defaults.js')
-  const sessionDataDefaultsFileExists = fs.existsSync(sessionDataDefaultsFile)
-
-  if (!sessionDataDefaultsFileExists) {
-    console.log('Creating session data defaults file')
-    if (!fs.existsSync(dataDirectory)) {
-      fs.mkdirSync(dataDirectory, { recursive: true })
-    }
-
-    fs.createReadStream(path.join(__dirname, '/lib/template.session-data-defaults.js'))
-      .pipe(fs.createWriteStream(sessionDataDefaultsFile))
-  }
-}
-
 console.log(`GOVV.UK Prototype Kit ${kitVersion}`)
 console.log('')
 console.log('starting...')
 
 ;(async () => {
-  createSessionDataDefaults()
```

Comment:
What's happening with the session data defaults file, is it going away entirely, or do we want to put it in the prototype starter files? Either way we should probably do something with `/lib/template.session-data-defaults.js` rather than leave it dangling 🤔

Reply:
it should be in the starter files

Reply:
I've now included it in the starter kit.  I've removed the documentation comment like we have in `filters.js`.

Reply:
Cool. Do we have a future ticket to look at the comments within starter files? Or are we planning to put all the documentation in the website?

Reply:
All the documentation should be on the website, otherwise if we change the interface we'd have to change the user's file too and I'm extremely keen to avoid editing files for users when they're upgrading within v13.